include/cpp-statsd-client/UDPSender.hpp

```diff
@@ -1,20 +1,39 @@
 #ifndef UDP_SENDER_HPP
 #define UDP_SENDER_HPP
 
+#ifdef _WIN32
+#define NOMINMAX
+#include <io.h>
+#include <winsock2.h>
+#include <ws2tcpip.h>
+#else
 #include <arpa/inet.h>
 #include <netdb.h>
 #include <sys/socket.h>
-#include <sys/types.h>
 #include <unistd.h>
+#endif
+
 #include <atomic>
 #include <cmath>
+#include <cstdint>
 #include <cstring>
 #include <deque>
 #include <mutex>
 #include <string>
 #include <thread>
 
 namespace Statsd {
+
+#ifdef _WIN32
+using SOCKET_TYPE = SOCKET;
+constexpr SOCKET_TYPE k_invalidSocket{INVALID_SOCKET};
+#define SOCKET_ERRNO WSAGetLastError()
+#else
+using SOCKET_TYPE = int;
+constexpr SOCKET_TYPE k_invalidSocket{-1};
+#define SOCKET_ERRNO errno
+#endif
```

Comment:
i found these helpful so i didnt have to do a bunch of `ifdef`ing inline in the code below